pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -71,6 +72,15 @@ const (
 )
 
 const (
+	cephOSDStart = `%s %s
+rc=$?
+if [ $rc -eq 0 ]; then
+	touch /tmp/osd-sleep
+	echo "OSD daemon exited with code 0, possibly due to OSD flapping. The OSD pod will sleep for 6 hrs. Restart the pod manually once the flapping issue is fixed"
+	sleep 21600s
```

Comment:
sleep has units for hours too
```suggestion
	sleep 6h
```